src/BulletManager.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <list>
+#include "Bullet.h"
+
+class BulletManager
+{
+public:
+  BulletManager() = default;
+  
+  ~BulletManager();
+  
+  void Fire();
+  std::list<TBulletPtr> GetAmmo() const;
```

Comment:
1) Yes, it describe the amount of bullets on a screen.
It has been fixed!

2) I am fully agree with you.
It has been fixed!